indico/modules/events/abstracts/templates/management/_abstract_list.html

```diff
@@ -56,7 +56,10 @@
                             {% for item in dynamic_columns %}
                                 <th class="i-table">{{ item.title }}</th>
                             {% endfor %}
-                            <th class="i-table thin-column"></th>
+                            {% if reviewed_abstracts %}
```

Comment:
Wouldn't `is not none` make more sense here? Otherwise you don't have the column if you are in the review area but haven't reviewed anything